linux-x86/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:latest
+FROM ubuntu:22.04
 
 ENV HOME /
 
 RUN apt-get update && apt-get install -y \
     build-essential \
     clang-tidy-12 \
-    cmake \
+    cmake>3.22 \
```

Comment:
I don't know the syntax here, but to me this looks like it sets the version to "greater than 3.22" when in fact it should set it to "equal 3.22".

Reply:
True, fixed by https://github.com/SpaceTeam/STS1_COBC_Docker/pull/6/commits/aa208b2a17552f7270f9307c1d74f68660e8f989 .